app/views/checkouts/ill_checkouts.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if illiad_loans.present? %>
+  <table class="table table-responsive-sm">
+    <div class='row pl-3 pt-2 pb-3'>
+      <h1>Interlibrary Loan Checkouts</h1>
+      <p class='pl-2 pt-2'>
+        (<a href='https://psu-illiad-oclc-org.ezaccess.libraries.psu.edu/illiad/upm/illiad.dll?Action=10&Form=10'>
```

Comment:
I wonder if this should be something like `Settings.illiad.manage_url` or something like this?

Reply:
yup will do that